components/stressPanel.js

```diff
@@ -78,30 +144,118 @@ export default function stressPanel(props) {
+                <div className="buttons are-small">
+                <button className="button is-link is-outlined " onClick={()=>{
+                  console.log("click test1")
+                  click = true;
+                   setParam({...params,minWorkgroups: 4,maxWorkgroups: 4, shufflePct: 0, barrierPct: 0, memStressPct: 0});
+                   props.params.minWorkgroups = 4;
+                   props.params.maxWorkgroups = 4;
+                   props.params.shufflePct = 0;
+                   props.params.barrierPct = 0;
+                   props.params.memStressPct = 0;
+                  // console.log("after click");
+                  // console.log(props.params)
+                  click = false;
+                }}>
+                  Test 1
+                </button> 
+                <button className="button is-link is-outlined " onClick={()=>{
+                  console.log("click test2")
+                  setParam({...params,minWorkgroups: 1024, maxWorkgroups: 1024, shufflePct: 100, barrierPct: 100, memStressPct: 0});
+                  click = true;
+                  props.params.minWorkgroups = 1024;
+                  props.params.maxWorkgroups = 1024;
+                  props.params.shufflePct = 100;
+                  props.params.barrierPct = 100;
+                  props.params.memStressPct = 0;
+                  click= false;
+                }}>
+                  Test 2
+                </button>
+                <button className="button is-link is-outlined " onClick={()=>{
+                  console.log("click test3")
+                  setParam({...params,minWorkgroups: 1024,maxWorkgroups: 1024, shufflePct: 100, barrierPct: 100, memStressPct: 100});
+                  click = true;
+                  props.params.minWorkgroups = 1024;
+                  props.params.maxWorkgroups = 1024;
+                  props.params.shufflePct = 100;
+                  props.params.barrierPct = 100;
+                  props.params.memStressPct = 100;
+                  click = false;
+                }}>
+                  Test 3
+                </button>
+                <button className="button is-link is-outlined " onClick={()=>{
+                   let maxWorkgroups =  Math.floor(Math.random() * (1024 - 4+1) + 4);
+                  let minWorkgroups = maxWorkgroups+1;
+                  while(minWorkgroups > maxWorkgroups){
+                    minWorkgroups =  Math.floor(Math.random() * (maxWorkgroups - 4+1) + 4);
+                  }
+                  let stressLineSize_ =  Math.floor(Math.random() * (128 - 1 + 1) + 1);
+                  let stressTargetLines_ =  Math.floor(Math.random() * (128 - 1 + 1) + 1);
+                  scratchMem = 4*stressLineSize_ * stressTargetLines_;
+                  setParam({minWorkgroups: minWorkgroups, 
+                            maxWorkgroups: maxWorkgroups, 
+                            testMemorySize: Math.floor(Math.random() * (4096 - 256 + 1) + 256),
```

Comment:
like `stressMemorySize`, we can make this fixed/dependent instead of randomly generating it